tmol/score/omega/omega_energy_term.py

```diff
@@ -38,10 +38,10 @@ def setup_block_type(self, block_type: RefinedResidueType):
         if hasattr(block_type, "omega_quad_uaids"):
             return
 
-        omega_quad_uaids = numpy.full((4, 3), -1, dtype=numpy.float32)
+        omega_quad_uaids = numpy.full((4, 3), -1, dtype=numpy.int32)
         if "omega" in block_type.torsion_to_uaids:
             omega_quad_uaids = numpy.array(
-                block_type.torsion_to_uaids["omega"], dtype=numpy.float32
+                block_type.torsion_to_uaids["omega"], dtype=numpy.int32
```

Comment:
Wrong data types.

Reply:
Ah, but at least you previously converted to int32 in setup_packed_block_types. (I had to figure out why you didn't also have to change the data type on the C++ side after making this float->int change)